internal/cli/atlas/logs/logs.go

```diff
@@ -19,7 +19,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func Builder() *cobra.Command {
+// MongoCLIBuilder is used in mongocli until atlascli is released.
+func MongoCLIBuilder() *cobra.Command {
```

Comment:
Since this will be deprecated in the future, I decided to make it have a different name and keep `Builder` for the atlascli. LMK if that's OK and if we can keep both in the same file or If we should create a new one. Thx!